### PR TITLE
Web Inspector: followup fixes after 254636@main

### PR DESCRIPTION
**Pull request by dcrousso**
#### 0bfbd6dc157bcb2aeb618c2f39762cb93549ac0e
<pre>
Web Inspector: followup fixes after 254636@main
<a href="https://bugs.webkit.org/show_bug.cgi?id=245385">https://bugs.webkit.org/show_bug.cgi?id=245385</a>

Reviewed by Patrick Angle.

* Source/WebInspectorUI/UserInterface/Base/Utilities.js:
(Set.prototype.some):
Fixed indentation.

* Source/WebInspectorUI/UserInterface/Models/ConsoleSnippet.js:
(WI.ConsoleSnippet.prototype.run):
Removed extra newline.

* Source/WebInspectorUI/UserInterface/Views/ConsoleSnippetTreeElement.js:
(WI.ConsoleSnippetTreeElement.prototype.onenter): Added.
(WI.ConsoleSnippetTreeElement.prototype.populateContextMenu): Added.
Support &quot;Enter&quot; to run and add a contextmenu item.

* Source/WebInspectorUI/UserInterface/Views/LogContentView.js:
(WI.LogContentView):
(WI.LogContentView.prototype._handleSnippetsNavigationItemContextMenu):
* Source/WebInspectorUI/UserInterface/Views/SourcesNavigationSidebarPanel.js:
(WI.SourcesNavigationSidebarPanel.prototype._populateCreateResourceContextMenu):
* Source/WebInspectorUI/Localizations/en.lproj/localizedStrings.js:
Fixed `&quot;...&quot;` in UI strings to be `&quot;\u2026&quot;`.

Canonical link: <a href="https://commits.webkit.org/254646@main">https://commits.webkit.org/254646@main</a>
</pre>

https://github.com/WebKit/WebKit/commit/839c5e37c65a17080d1a6da0018d69b98458bff8

| Misc | iOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/6/builds/89762 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/69/builds/34313 "Built successfully") | [  ~~🛠 mac~~](https://ews-build.webkit.org/#/builders/43/builds/20466 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🛠 wpe~~](https://ews-build.webkit.org/#/builders/8/builds/99097 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🛠 🧪 win~~](https://ews-build.webkit.org/#/builders/10/builds/155914 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") 
| | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/64/builds/32807 "Built successfully") | [  ~~🛠 mac-debug~~](https://ews-build.webkit.org/#/builders/71/builds/28260 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🛠 gtk~~](https://ews-build.webkit.org/#/builders/36/builds/82121 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🛠 wincairo~~](https://ews-build.webkit.org/#/builders/12/builds/93447 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/19/builds/95409 "Passed tests") | [  ~~🧪 ios-wk2~~](https://ews-build.webkit.org/#/builders/68/builds/26074 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🛠 mac-AS-debug~~](https://ews-build.webkit.org/#/builders/61/builds/76600 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🧪 gtk-wk2~~](https://ews-build.webkit.org/#/builders/36/builds/82121 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | 
| | [  ~~🧪 api-ios~~](https://ews-build.webkit.org/#/builders/9/builds/80951 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🧪 api-mac~~](https://ews-build.webkit.org/#/builders/43/builds/20466 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🧪 api-gtk~~](https://ews-build.webkit.org/#/builders/36/builds/82121 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | 
| | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/66/builds/30558 "Built successfully") | [  ~~🧪 mac-wk1~~](https://ews-build.webkit.org/#/builders/43/builds/20466 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/67/builds/30307 "Built successfully") | [  ~~🧪 mac-wk2~~](https://ews-build.webkit.org/#/builders/43/builds/20466 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | | 
| | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/63/builds/33758 "Built successfully") | [  ~~🧪 mac-AS-debug-wk2~~](https://ews-build.webkit.org/#/builders/61/builds/76600 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | | 
| [✅ 🛠 🧪 unsafe-merge](https://ews-build.webkit.org/#/builders/75/builds/1381 "Built successfully and passed tests") | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/65/builds/32469 "Built successfully") | [  ~~🧪 mac-wk2-stress~~](https://ews-build.webkit.org/#/builders/43/builds/20466 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | |

### CODE REVIEW

File: Source/WebInspectorUI/UserInterface/Views/ConsoleSnippetTreeElement.js [hunk 1]
```diff
@@ -54,6 +54,13 @@ WI.ConsoleSnippetTreeElement = class ConsoleSnippetTreeElement extends WI.Script
         return true;
     }
 
+    onenter()
```
Comment (JosephPecoraro) [inline Source/WebInspectorUI/UserInterface/Views/ConsoleSnippetTreeElement.js:57]: I wonder if there are other things that should have an `onenter` that matches `onspace`.

Namely: `BootstrapScriptTreeElement`, `LocalResourceOverrideTreeElement`. 🤔

File: Source/WebInspectorUI/UserInterface/Views/ConsoleSnippetTreeElement.js [hunk 2]
```diff
@@ -69,6 +76,20 @@ WI.ConsoleSnippetTreeElement = class ConsoleSnippetTreeElement extends WI.Script
         return super.canSelectOnMouseDown(event);
     }
 
+    populateContextMenu(contextMenu, event)
+    {
+        contextMenu.appendSeparator();
+
+        contextMenu.appendItem(WI.UIString("Run Console Snippet"), () => {
+            WI.auditManager.editing = true;
```
Comment (JosephPecoraro) [inline Source/WebInspectorUI/UserInterface/Views/ConsoleSnippetTreeElement.js]: Err, audit manager?

Comment (dcrousso) [reply Source/WebInspectorUI/UserInterface/Views/ConsoleSnippetTreeElement.js]: oops bad copypasta

**Comment by JosephPecoraro** [comment]
Looks good to me!

File: Source/WebInspectorUI/UserInterface/Views/ConsoleSnippetTreeElement.js (hunk 1 above)
Comment (dcrousso) [reply Source/WebInspectorUI/UserInterface/Views/ConsoleSnippetTreeElement.js:57]: probably?  i personally see a bit of difference between space and enter, but maybe that's just me

**Review by patrickangle — APPROVED** [review]

**Pull request merged**